tips-tricks/TroubleshootingandTips.rst

```diff
@@ -667,3 +667,32 @@ creation, you can set the SerDe and properties:
 Notice the double escape required!
 
 .. |image1| image:: ../media/kylo-troubleshooting/2_executesparkjob.png
+
+GetTableData vs ImportSqoop Processor
+========================================================================
+
+Problem
+-------
+
+You need to load data from a structured datastore.
+
+Solution
+--------
+
+There are two major Nifi processors for importing data into Kylo (i.e. into Hadoop): GetTableData and ImportSqoop.
```

Comment:
Please change Nifi to NiFi to reflect standard project name. Please update all such uses.